package.json

```diff
@@ -83,9 +86,6 @@
     "stylelint-config-recommended-scss": "^6.0.0",
     "stylelint-config-standard": "^24.0.0",
     "stylelint-config-standard-scss": "^3.0.0",
-    "vibe-storybook-components": "^0.1.0",
-    "postcss": "8.4.4",
-    "postcss-scss": "^4.0.3",
-    "postcss-value-parser": "4.1.0"
+    "vibe-storybook-components": "^0.1.1-export-storybook-css-tokens-e593d3c203.0"
```

Comment:
Replace with normal version when this is released https://github.com/mondaycom/vibe-storybook-components/pull/17